src/components/JUPAssetSearchBox/JUPAssetSearchBox.tsx

```diff
@@ -70,8 +72,7 @@ const JUPAssetSearchBox: React.FC<IJUPAssetSearchBoxProps> = ({ fetchFn }) => {
     <Autocomplete
       freeSolo
       options={searchBoxResults.map((searchBoxValue) => searchBoxValue)}
-      // MUST: fix magic number? currently splits the value and takes the asset id portion (first index), might be able to do this more cleanly
-      onChange={(e, value: any) => value && fetchFn(value.split("-")[1].trim())}
+      onChange={(e, value) => value && fetchFn(value.split("-")[AssetIdIndex].trim())}
```

Comment:
fyi, index 1 is not the first index it's the 2nd. 0 is the first index in an array.

Reply:
all good, working as intended